KoansAndKatas/koanwiki.md

```diff
@@ -0,0 +1,59 @@
+
+###Solution presentation
+The host will select one volunteer to present their solution (for up to 5 minutes). If nobody volunteers, the host will present his or her own solution to the meeting. The main purpose of this presentation will be to explain not so much the solution itself, but the concepts utilized in the solution.
+
+###Q&A   
+For up to 5 minutes after the presentation, there will be opportunities for questions to be asked. Anyone can ask a question and anyone is free to answer them.  
+
+###Meeting archive
+The host will archive all notes from the meeting into a Company Google Drive folder (within the Engineering folder). The file will be in Google Docs format. The naming format of the file will be "YYYY-MM-DD - Problem Topic}".
+
+###Task assignments
+Optionally, the sessions could coexist with items that are part of the current sprint. To get a deeper understanding of the topics, somebody on the team who has expressed interest in getting further practice, could be assigned to a story or issue which uses those topics.
+
+###Example problems
+Here are some examples of challenges that others have come up with. I'm not sure if it would serve us best to use any of these. Our time to meet together is limited and we should choose our topics carefully, to get the most bang for our buck. Additionally, the existing challenges I've found aren't necessarily designed for the sake of teaching specific concepts, but rather for solo practice. I've listed the challenges below for the sake of example. 
+
+**Beginner**  
+https://github.com/ChrisMarinos/FSharpKoans
+
+**Intermediate**  
+http://ocaml.org/learn/tutorials/99problems.html
+https://github.com/c4fsharp/Dojo-Fractal-Forest
+
+**Difficult**  
+https://github.com/c4fsharp/Dojo-Type-Provider-Treasure-Hunt (long)
+https://github.com/c4fsharp/Dojo-Canopy-2048 (long)
+
+**Machine-Learning**  
+https://github.com/c4fsharp/Dojo-Ham-Or-Spam
+https://github.com/c4fsharp/Dojo-Digits-Recognizer 
+https://github.com/c4fsharp/Dojo-Markov-Bot
+
+**Other**  
+https://github.com/c4fsharp/Dojo-Picasquez-and-Velasso (not a strong F# tutorial but allows for creativity)
+
+
```

Comment:
Looks good to me. @HoraceGonzalez ?

First step would be to setup the calendar invites, and get some volunteers for our first problem or two!